python/pyproject.toml

```diff
@@ -100,3 +100,6 @@ cmake.minimum-version = "3.26.4"
 ninja.make-fallback = true
 sdist.reproducible = true
 wheel.packages = ["ucxx"]
+
+[tool.codespell]
+ignore-words-list = "cancelation,inflight"
```

Comment:
Reasoning:
- cancelation: codespell corrects it to the British English spelling "cancellation" (with two `l`s), whereas we chose the American spelling "cancelation` (with one `l`) as it is the one used by UCX;
- inflight: that's how we've been writing it and there are many instances of the word "inflight" online, although dictionaries seem to point "in-flight" as correct `codespell` didn't find all its instances automatically so for now I ignored it, but we can review it if the consensus is we should definitely switch to "in-flight".